lib/super_digit.rb

```diff
@@ -1,15 +1,15 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) where n is the number of digits. as the number doubles, worst case, length of number also doubles. 5*2=10
+# Space Complexity - O(1)
 def super_digit(n)
```

Comment:
Because `n.digits.dum` is O(n) in time complexity, this is O(n^2) in time complexity and O(n) in space complexity (recursion)